### DISCUSSION TITLE
Microsoft SQL Server data source connector | NTLM authentication

### DISCUSSION THREAD

**Discussion by rickygi** [prisma/prisma#20239] [Q&A] [answered]
Extend Microsoft SQL Server data source connector to support NTLM authentication.

[Microsoft SQL Server data source connector](https://www.prisma.io/docs/concepts/database-connectors/sql-server#using-integrated-security-windows-only)

The following example uses a specific Active Directory user to log in to Microsoft SQL Server:

`sqlserver://localhost:1433;initialCatalog=sample;integratedSecurity=true;username=prisma;password=aBcD1234;trustServerCertificate=true;`

[SQL Server | Setting the connection properties](https://learn.microsoft.com/en-us/sql/connect/jdbc/setting-the-connection-properties?view=sql-server-ver16&viewFallbackFrom=sql-server-ver19)

`authenticationScheme=NTLM`

**Comment by janpio** [comment]
That sounds like a feature request you better open as an explicit feature request issue. Thanks.

**Reply by nurul3101** [reply]
Please let us know once you had a chance to submit a Feature Request, you can use this link to submit one:

https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=

**Reply by quietcoding** [reply, marked as answer]
Thanks @nurul3101, we've created a feature request: https://github.com/prisma/prisma/issues/20265 for this. Let us know if there are more questions regarding this request.

**Reply by nurul3101** [reply]
Thank you so much for creating the feature request :)
